src/QMCHamiltonians/StressPBC.h

```diff
@@ -0,0 +1,111 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
+//
+// File developed by: Raymond Clay III, j.k.rofling@gmail.com, Lawrence Livermore National Laboratory
+//                    Mark A. Berrill, berrillma@ornl.gov, Oak Ridge National Laboratory
+//                    Yubo Paul Yang, yyang173@illinois.edu, University of Illinois Urbana-Champaign
+//
+// File created by: Raymond Clay III, j.k.rofling@gmail.com, Lawrence Livermore National Laboratory
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+#ifndef QMCPLUSPLUS_STRESSPBC_HAMILTONIAN_H
+#define QMCPLUSPLUS_STRESSPBC_HAMILTONIAN_H
+#include "QMCHamiltonians/ForceBase.h"
+#include "LongRange/LRCoulombSingleton.h"
+#include "OhmmsPETE/SymTensor.h"
+
+namespace qmcplusplus
+{
+struct StressPBC : public OperatorBase, public ForceBase
+{
+  typedef LRCoulombSingleton::LRHandlerType LRHandlerType;
+
+  SymTensor<RealType, OHMMS_DIM> stress_ee_const;
+  SymTensor<RealType, OHMMS_DIM> stress_eI_const;
+
+  // need wave function for kinetic stress
+  TrialWaveFunction& Psi;
+
+  ///source particle set
+  ParticleSet& PtclTarg;
+  ParticleSet& PtclA;
+  ///long-range Handler
+  LRHandlerType* AA;
```

Comment:
Can we leverage unique_ptr or shared_ptr for all the raw pointers in the added class?

Reply:
Yes, addressed.